docs/_templates/sidebar-version.html

```diff
@@ -1 +1,2 @@
-<div class="version centered"><i>version {{ version }}</i></div>
+<div class="version centered"><i>version {%- include 'version-switcher.html' -%}</i></div>
```

Comment:
Can you make it so the spacing between the word and the dropdown is always equal to one space? Right now, it stretches depending on the width of the sidebar, and usually looks either too wide or too narrow.